person.rb

```diff
@@ -1,3 +1,5 @@
+require './corrector'
+
 class Person
```

Comment:
- [ ] Please keep in mind that the `Person` class has to inherit from the `Namabale` class, and in this case, I don't see the `Namabale` class in your files, unless it is the `Corrector` class!

Reply:
Thank you for your suggestion. I have implemented all the changes told by you.

Reply:
This issue is well resolved! ✔️